erpnext/projects/doctype/task/task.py

```diff
@@ -143,6 +144,7 @@ def on_update(self):
 		self.populate_depends_on()
 		self.notify()
 		self.assign_todo()
+		update_project_estimated_time(self)
```

Comment:
move this to after_save hook so that the save doesn't slow down because of this

Reply:
used after_save method to update estimated_time in project doctype